repo/currency_converter.go

```diff
@@ -0,0 +1,93 @@
+package repo
+
+import (
+	"errors"
+	"math/big"
+	"strings"
+
+	"github.com/OpenBazaar/wallet-interface"
+)
+
+var (
+	// ErrPriceCalculationRequiresExchangeRates - exchange rates dependency err
+	ErrPriceCalculationRequiresExchangeRates = errors.New("can't calculate price with exchange rates disabled")
+)
+
+// CurrencyConverter
+type CurrencyConverter struct {
+	ReserveWallet wallet.Wallet
+}
+
+func NewCurrencyConverter(reserveWallet wallet.Wallet) CurrencyConverter {
+	return CurrencyConverter{
+		reserveWallet,
```

Comment:
We should always label field names in explicit structs so the ordering is non-specific and the constructions are well-described instead of having to lookup the struct def.

Reply:
Okay @placer14 but I believe there's only one attribute of the struct to begin with...

Reply:
![image](https://user-images.githubusercontent.com/45482/73756559-c6100980-4735-11ea-82d6-ece1bc3aa974.png)

Reply:
Right. So we should have:
```
CurrencyConverter{
  ReserveWallet: reserveWallet,
}
```

This does two things for us in the future:
1. We know exactly what this is used as within CurrencyConverter without looking at the definition (we aren't always looking at code inside an IDE)
2. We can expand this struct without any additional noise in future changes (extra fields don't force us to change this line again)
3. You can be less verbose with the function argument name

Minor wins which all add up.